.travis.yml

```diff
@@ -102,17 +102,23 @@ jobs:
       script:
         - tox -e py37
 
+    - stage: DeployPyPI
+      python: 3.7
+      if: tag IS present
```

Comment:
Is this needed in addition to `on` below (or the other way around, can we leave out `on`?)

Reply:
We can leave the `on` out but it does not hurt. 
With `if: tag IS present` it will not show/run if not needed in Travis.

Can change as you prefer.

Reply:
Then remove the `on` please. I'm pretty sure we'll stumble across it otherwise in a couple months from now and wonder why it was there.